packages/formation/sass/modules/_m-nav-sidebar.scss

```diff
@@ -206,7 +206,7 @@ $level-3-hover-padding: 8px 12px 8px 30px;
       }
 
       &:focus {
-        outline: 2px solid $color-gold;
+        outline: $focus-outline;
```

Comment:
I don't think this is actually used anywhere, but I felt like removing it was out of scope for the PR. I updated it just to keep things consistent.

https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/ce8e5ff5d58eeeb3e0cbafed295ef6ccce2528a1/packages/formation/sass/modules/_m-nav-sidebar.scss#L198-L212

![Unused in `content-build`](https://user-images.githubusercontent.com/2008881/169139391-979c05d7-b952-4532-a9eb-6c40a3a2b4c7.png)

![Unused in `vets-website`](https://user-images.githubusercontent.com/2008881/169139452-8b8d5575-7695-41ef-890c-095959d4f852.png)

![Unused in `component-library` react components](https://user-images.githubusercontent.com/2008881/169139572-4721fbd5-ff53-4be3-a919-eceda7d53427.png)

It's even unused in the [`sidenav.js` file](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/ce8e5ff5d58eeeb3e0cbafed295ef6ccce2528a1/packages/formation/js/sidenav.js):

![Unused in formation js](https://user-images.githubusercontent.com/2008881/169139918-ff010c30-7031-4a1c-a404-2173bab66089.png)